gdal/doc/source/drivers/raster/vrt.rst

```diff
@@ -796,24 +796,95 @@ Default Pixel Functions
-- **imag**: extract imaginary part from a single raster band (0 for non-complex)
-- **complex**: make a complex band merging two bands used as real and imag values
-- **mod**: extract module from a single raster band (real or complex)
-- **phase**: extract phase from a single raster band [-PI,PI] (0 or PI for non-complex)
-- **conj**: computes the complex conjugate of a single raster band (just a copy if the input is non-complex)
-- **sum**: sum 2 or more raster bands
-- **diff**: computes the difference between 2 raster bands (b1 - b2)
-- **mul**: multiply 2 or more raster bands
-- **cmul**: multiply the first band for the complex conjugate of the second
-- **inv**: inverse (1./x). Note: no check is performed on zero division
-- **intensity**: computes the intensity Re(x*conj(x)) of a single raster band (real or complex)
-- **sqrt**:perform the square root of a single raster band (real only)
-- **pow**: raise a single raster band to a constant power, specified with argument "power" (real only)
-- **log10**: compute the logarithm (base 10) of the abs of a single raster band (real or complex): log10( abs( x ) )
-- **dB**: perform conversion to dB of the abs of a single raster band (real or complex): 20. * log10( abs( x ) )
-- **dB2amp**: perform scale conversion from logarithmic to linear (amplitude) (i.e. 10 ^ ( x / 20 ) ) of a single raster band (real only)
-- **dB2pow**: perform scale conversion from logarithmic to linear (power) (i.e. 10 ^ ( x / 10 ) ) of a single raster band (real only)
+
+.. list-table::
+   :widths: 15 10 20 55
+   :header-rows: 1
+
+   * - PixelFunctionType
+     - Number of input sources
+     - PixelFunctionArguments
+     - Description
+   * - **cmul**
+     - 2
+     - -
+     - multiply the first band for the complex conjugate of the second
+   * - **complex**
+     - 2
+     - -
+     - make a complex band merging two bands used as real and imag values
+   * - **conj**
+     - 1
+     - -
+     - computes the complex conjugate of a single raster band (just a copy if the input is non-complex)
+   * - **dB**
+     - 1
+     - -
+     - perform conversion to dB of the abs of a single raster band (real or complex): ``20. * log10( abs( x ) )``
+   * - **dB2amp**
+     - 1
+     - -
+     - perform scale conversion from logarithmic to linear (amplitude) (i.e. ``10 ^ ( x / 20 )`` ) of a single raster band (real only)
+   * - **dB2pow**
+     - 1
+     - -
+     - perform scale conversion from logarithmic to linear (power) (i.e. ``10 ^ ( x / 10 )`` ) of a single raster band (real only)
+   * - **diff**
+     - 2
+     - -
+     - computes the difference between 2 raster bands (``b1 - b2``)
+   * - **imag**
+     - 1
+     - -
+     - extract imaginary part from a single raster band (0 for non-complex)
+   * - **intensity**
+     - 1
+     - -
+     - computes the intensity ``Re(x*conj(x))`` of a single raster band (real or complex)
+   * - **interpolate_exp**
+     - >= 2
+     - ``x0``, ``dx``, ``x``
```

Comment:
My mind had some trouble with the x naming, confusing it with the horizontal axis. Perhaps ``t`` would make it clearer than this is not interpolation along an axis of the 2D raster plan ? (if so, the change should be made in the code too)

Reply:
Switched to `t` throughout